port_ocean/config/settings.py

```diff
@@ -68,6 +68,7 @@ class IntegrationConfiguration(BaseOceanSettings, extra=Extra.allow):
     initialize_port_resources: bool = True
     scheduled_resync_interval: int | None = None
     client_timeout: int = 60
+    use_provisioned_defaults: bool = False
```

Comment:
are we sure we want to use only two options `False` and `True` ? maybe worth changing the variable to `provisioning_mode` 
- port
- local
which will allow us more flexability in the future?

Reply:
WDYM? Why would that be different?